docs/guides/run-nwaku-node.md

```diff
@@ -34,17 +38,8 @@ Once you have built the `nwaku` node, run it using the default configuration:
 ./build/wakunode2 --help
 ```
 
-By default, a `nwaku` node is configured to do the following:
-
-- Generate a new private key and `PeerID`.
-- Listen for incoming libp2p connections on the default TCP port (`60000`).
-- Subscribe to the default Pub/Sub topic (`/waku/2/default-waku/proto`).
-- Start the `JSON-RPC` HTTP server on the default port (`8545`).
-- Enable the `Relay` protocol for relaying messages.
-- Enable the `Store` protocol as a client, allowing it to query peers for historical messages but not store any message itself.
-
-:::info
-For more advanced configurations like enabling other protocols or maintaining a consistent `PeerID`, please refer to the [Configuration Methods](https://github.com/waku-org/nwaku/blob/master/docs/operators/how-to/configure.md) guide.
+:::tip
+For more advanced configurations like enabling other protocols or maintaining a consistent `PeerID`, please refer to the [Node Configuration Methods](/guides/reference/node-config-methods) guide.
 :::
 
 ## Connect the Node
```

Comment:
There are two part to this topic:

1. Bootstrapping to the Waku network
2. Discovering more peer

DNS Discovery and static peers solve (1)
Discv5 solves (2)

However, can't solve (2) without solving (1). I think this section should focus on (1) and be called "Bootstrapping" the node or something like that.

Then, second section can be  on "peer discovery" 

Both section having a single liner on what bootstrap and peer discovery are.